Applications/_Plugins/Common/gltf/Vulkan/GltfTechnique.cpp

```diff
@@ -20,11 +20,16 @@
 
 #include "GltfTechnique.h"
 
+#include "GltfCommon.h"
+
+#include "DynamicBufferRingVK.h"
+#include "StaticBufferPoolVK.h"
+
 void GltfTechnique::Draw(VkCommandBuffer cmd_buf)
 {
-    CMP_DWORD dwNodesVisible;
+    std::uint32_t dwNodesVisible;
```

Comment:
It's super important to try and use `uint32_t` and other things in the standard library instead of Windows primitves. This is the expectation from vulkan and leads to greater cross platform compatibility. Just calling that out here but applies to all other places where this was updated

Reply:
@UebelAndre  Type defs are been updated (Takes time) to use one consistent definition, CMP_Core uses CGU_ types and others use CMP_  (mixed in with std:..) which are to be replaced by CGU_  types eventually. 

CGU types are designed for use on multiple frameworks and compilers. (ISPC, GCC, MS, etc) and Shader compilers for (OpenCL, DirectX, Vulkan, ..)